### ISSUE TITLE
Can someone explain train options?

### ISSUE THREAD

**Issue by bob0110** [YunYang1994/tensorflow-yolov3#272]
Can someone explain these to me?
What is the difference between first and second stage?
__C.TRAIN.WARMUP_EPOCHS         = 2
__C.TRAIN.FISRT_STAGE_EPOCHS    = 20
__C.TRAIN.SECOND_STAGE_EPOCHS   = 30

Is there a ratio in numbers you should follow to get good results?

**Comment by Kasthooriraja** [comment]
__C.TRAIN.WARMUP_EPOCHS deals with the learning rate changes. there are two learning rate config params
1. __C.TRAIN.LEARN_RATE_INIT       = 1e-4
2.__C.TRAIN.LEARN_RATE_END        = 1e-6
up to the WARMUP_EPOCHS the learning rate increases to LEARN_RATE_INIT from 0
after that lr reac
